framework/src/executioners/Eigenvalue.C

```diff
@@ -58,4 +58,34 @@ Eigenvalue::execute()
   Moose::SlepcSupport::slepcSetOptions(_eigen_problem, _pars);
 #endif
   Steady::execute();
+
+  // print eigenvalues
+  std::ostringstream ss;
+  ss << '\n';
+  ss << "*******************************************************\n";
+  ss << " Eigenvalues";
+  ss << std::fixed << std::setprecision(10);
+  auto & eigen_sys = _eigen_problem.getNonlinearEigenSystem();
+  auto & eigenvalues = eigen_sys.getAllConvergedEigenvalues();
+  if (eigenvalues.size() == 1)
+  {
+    auto & eigenvalue = *eigenvalues.begin();
+    ss << " = " << eigenvalue.first;
+    if (eigenvalue.second != 0)
+      ss << "+" << eigenvalue.second << "i";
+    ss << "\n";
+  }
+  else
+  {
+    ss << ":\n";
+    for (auto & eigenvalue : eigenvalues)
+    {
+      ss << "   " << eigenvalue.first;
+      if (eigenvalue.second != 0)
+        ss << "+" << eigenvalue.second << "i";
+      ss << "\n";
+    }
+  }
+  ss << "*******************************************************";
```

Comment:
We do not want to do this. Suppose we have a lot of eigenvalues, then we won't print it out on screen.

Reply:
Almost for sure less than 100. This gives better user experience. The eigenvalues is most of interest and users want them to be printed explicitly.